bcparis-service/bcparis-service.properties

```diff
@@ -44,7 +44,7 @@ endpoint.icbc.rest.header.password=${ENDPOINT_ICBC_REST_HEADER_PASSWORD}
 endpoint.iamp-email-service.rest.path=/api/v1/emails
 endpoint.iamp-email-service.rest.username=${ENDPOINT_IAMP_EMAIL_SERVICE_REST_USERNAME}
 endpoint.iamp-email-service.rest.password=${ENDPOINT_IAMP_EMAIL_SERVICE_REST_PASSWORD}
-endpoint.iamp-email-service.email.to=marcos.agostta@opticca.com
+endpoint.iamp-email-service.email.to=no_reply@gov.bc.ca
```

Comment:
do we have a service account for justice that we can use for this? or should we submit to chris/me